### PR TITLE
Dashboard overview

### PR DESCRIPTION
**Pull request by BastinJafari**
I will put the tutorial on how to connect with JinaD into the next PR

### CODE REVIEW

File: chapters/dashboard/index.md [hunk 1]
```diff
@@ -1,87 +1,24 @@
-# Using Dashboard to Monitor Logs and View Flow
+# Jina Dashboard
 
-**Dashboard** (beta) offers the insight of running tasks in Jina. With dashboard, one can analyze logs, design flows and view Jina Hub images.
+**Jina Dashboard** is a visual interface to facilitate Development with Jina.
```
Comment (FionnD) [inline chapters/dashboard/index.md]: You might consider replacing this with the new description Lucia came up with for our website. Not a requirement, just to keep [brandalignment](https://docs.google.com/document/d/1DVSUMRye5icmtwGCRBpwmvKgKtZOiczdVSH0NRg6xvQ/edit#)

File: chapters/dashboard/index.md [hunk 2]
```diff
@@ -1,87 +1,24 @@
...
 
-![flow demo](flow-demo.gif)
+In order do use the Dashboard visit https://dashboard.jina.ai
```
Comment (FionnD) [inline chapters/dashboard/index.md]: In order **to** use

File: chapters/dashboard/index.md [hunk 3]
```diff
@@ -1,87 +1,24 @@
...
-- Grouping logs by Pods, Executors. Full text search on logs.
-- Drag & drop flow design, setting properties of each Pod via a webform.
-- Flow can be imported from/exported to YAML.
+![flow demo](overview.gif)
 
 
+# Features
 
-## Start the Log Server
+-  ##  [Monitoring of logs](log-stream.md)
```
Comment (FionnD) [inline chapters/dashboard/index.md]: Is the **of**  necessary?

File: chapters/dashboard/index.md [hunk 4]
```diff
@@ -1,87 +1,24 @@
...
+-  ##  [Design and start Flows with Drag & Drop](flow.md)
```
Comment (FionnD) [inline chapters/dashboard/index.md]: I wouldn't mix **and** with **&**. I'd just use one.

File: chapters/dashboard/log-stream.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
+## Start the Log Server
```
Comment (FionnD) [inline chapters/dashboard/log-stream.md]: For this page, can you try using this template? https://github.com/jina-ai/docs/pull/21 

I know your approach is simpler, but it's an attempt to have a standard look throughout the docs.

File: chapters/dashboard/self-host.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+## Self-host a Dashboard
```
Comment (FionnD) [inline chapters/dashboard/self-host.md]: For this page, can you try using this template? https://github.com/jina-ai/docs/pull/21 

I know your approach is simpler, but it's an attempt to have a standard look throughout the docs.

**Review by FionnD — CHANGES_REQUESTED** [review]

File: chapters/dashboard/self-host.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+One can self-host a dashboard locally.
+
+1. `git clone https://github.com/jina-ai/dashboard.git && cd dashboard`.
+2. Install dependencies using command `yarn`.
+3. Run dashboard via the following ways .
+
+### Run in the debug mode
+
+1. `node testServer`
```
Comment (Roshanjossey) [inline chapters/dashboard/self-host.md]: We have other scripts for this now. 

`npm run dev`

runs both dev server and react app in parallel.

So, I guess we have to make a call on telling users the easiest way, explain different parts of it or both.

File: chapters/dashboard/self-host.md (hunk 6 above)
Comment (Roshanjossey) [reply chapters/dashboard/self-host.md]:
```suggestion
## Self-host Dashboard
```

**Review by FionnD — APPROVED** [review]

**Pull request merged**